src/styles/description.scss

```diff
@@ -0,0 +1,39 @@
+.description {
+  display: grid;
+  grid-template-columns: repeat(2, 510px);
+  justify-content: center;
+
+  &__title {
+    font-weight: 600;
+    font-size: 48px;
+    line-height: 48px;
+    letter-spacing: -1.5px;
+    color: #000;
```

Comment:
create variables for colors
colorWhite
colorBlack
colorBlue

Reply:
fixed